src/style.css

```diff
@@ -0,0 +1,68 @@
+body {
+  background-color: rgb(234, 230, 230);
+}
+
+.content {
+  border: 2px solid rgb(184, 220, 280);
+  width: 70%;
+  margin: 20px auto;
+  padding: 5px;
+  border-radius: 5px;
+  box-shadow: rgba(0, 0, 0, 0.35) 0 5px 15px;
+}
+
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-around;
+}
+
+form {
+  border: 2px solid gainsboro;
+  width: 70%;
+  margin: auto;
+  padding: 8px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  gap: 8px;
+}
+
+input {
+  width: 80%;
+  height: 30px;
+  font-size: 17px;
+  font-style: italic;
+}
+
+ul {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.bottom {
+  padding: 5px 30px;
+  width: 100%;
+  font-size: 17px;
+  font-weight: 600;
+}
+
+.toDoDivision {
+  /* border: 2px solid yellow; */
+  display: flex;
+  align-items: center;
+  justify-content: space-around;
+  min-width: 550px;
+}
+
+.checkbox {
+  border: 2px solid red;
+  width: 5px;
+  margin-left: 1px;
+  padding-left: 0%;
+}
```

Comment:
- [ ] According to the requirements, Your list should be a clone of the part of the minimalist project captured in [the video](https://www.youtube.com/watch?v=AcUd-_Yjjqg), kindly update it 💯 

| Your Design  | Original Design  |   
| --- | --- |
| <img src="https://user-images.githubusercontent.com/45231276/192723670-131ef373-b1c8-4368-8864-99eb193df639.PNG" width="500" />  | <img src="https://user-images.githubusercontent.com/45231276/192723817-4e4a9186-1fed-44cf-89d2-cdbe416abe7d.png" width="500" /> |

Reply:
Well noted